.github/scripts/update_layer_arn.sh

```diff
@@ -65,8 +66,33 @@ for file in $files; do
 
         # Replace all the "prefix_pseudo_region"'s in the file
         # prefix_pseudo_region:\d+ ==> line_pseudo_region
-        sed -i -e "s/$prefix_pseudo_region:[[:digit:]][[:digit:]]*/$line_pseudo_region/g" docs/index.md
+        sed -i -e "s/$prefix_pseudo_region:[[:digit:]][[:digit:]]*/$line_pseudo_region/g" $DOCS_FILE
       done
     fi
   done
 done
+
+echo "[+] Finished processing all commercial regions"
+
+# Now we need to process GovCloud regions
+# 
+# GovCloud layers are not available in the CDK output files, but we know the ARN format and that the version is the same
+# as the one in eu-central-1. So we can optimistically update the version of the GovCloud layers in the documentation.
+#
+# The GovCloud ARNs are (note that the account IDs are different in both):
+# arn:aws-us-gov:lambda:us-gov-west-1:165093116878:layer:AWSLambdaPowertoolsTypeScriptV2:25
+# arn:aws-us-gov:lambda:us-gov-east-1:165087284144:layer:AWSLambdaPowertoolsTypeScriptV2:25
+
+version=$(echo "$line" | cut -d ':' -f 8) # version = 25
+arn_us_gov_west_1="arn:aws-us-gov:lambda:us-gov-west-1:165093116878:layer:AWSLambdaPowertoolsTypeScriptV2:$version"
+prefix_us_gov_west_1=$(echo "$arn_us_gov_west_1" | cut -d ':' -f 1-7)
+arn_us_gov_east_1="arn:aws-us-gov:lambda:us-gov-east-1:165087284144:layer:AWSLambdaPowertoolsTypeScriptV2:$version"
+prefix_us_gov_east_1=$(echo "$arn_us_gov_east_1" | cut -d ':' -f 1-7)
+echo -e "\t[*] ARN GovCloud US West 1: $arn_us_gov_west_1"
+echo -e "\t[*] ARN GovCloud US East 1: $arn_us_gov_east_1"
```

Comment:
You could remove all of this here by passing the layer number as a variable in the workflow.